config/pq_capablities.ini

```diff
@@ -1,83 +1,89 @@
 [Brightness]
```

Comment:
who uses this ini file? yaml is much better?

Reply:
these capability files will help us enable the features on yaml file. We are just using for reference.